### PR TITLE
Internal: Release based on Tagged - include "dev" edition (Core) [ED-10152]

### PR DESCRIPTION
**Pull request by maork-elementor**

### CODE REVIEW

File: .github/workflows/release-dev-from-beta/action.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    description: 'The name of the plugin.'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Replace Release Name In Plugin File With Dev
+      shell: bash
+      run: |
+            unzip -o ${{ inputs.BUILD_ZIP_FILE_PATH }} ${{ inputs.PLUGIN_NAME }}/${{ inputs.PLUGIN_NAME }}.php
```
Comment (yotamselementor) [inline .github/workflows/release-dev-from-beta/action.yml:18]: Extract `${{ inputs.PLUGIN_NAME }}/${{ inputs.PLUGIN_NAME }}.php` to a variable

File: .github/workflows/release-dev-from-beta/action.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            # Remove rc
+            sed -i -E "s/Version: .*/Version: ${RELEASE_NAME}/g" $PLUGIN_FILE_PATH
+            sed -i -E "s/ELEMENTOR_VERSION', '.*'/ELEMENTOR_VERSION', '${RELEASE_NAME}'/g" $PLUGIN_FILE_PATH
+
+            # Add the updated plugin file to the zip archive
+            zip -u ${{ inputs.BUILD_ZIP_FILE_PATH }} $PLUGIN_FILE_PATH -p ${{ inputs.PLUGIN_NAME }}/${{ inputs.PLUGIN_NAME }}.php
+
+            # Set zip file name to dev, and remove '-rc'
+            CURRENT_BUILD=${{ inputs.BUILD_ZIP_FILE_PATH }}
+            DEV_BUILD_ZIP_FILE_PATH=$(echo "$CURRENT_BUILD" | sed 's/beta/dev/' | sed 's/-rc//')
```
Comment (yotamselementor) [inline .github/workflows/release-dev-from-beta/action.yml:34]:
```suggestion
            DEV_BUILD_ZIP_FILE_PATH=$(echo "${{ inputs.BUILD_ZIP_FILE_PATH }}" | sed 's/beta/dev/' | sed 's/-rc//')
```

File: .github/workflows/release-dev-from-beta/action.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  steps:
+    - name: Replace Release Name In Plugin File With Dev
+      shell: bash
+      run: |
+            unzip -o ${{ inputs.BUILD_ZIP_FILE_PATH }} ${{ inputs.PLUGIN_NAME }}/${{ inputs.PLUGIN_NAME }}.php
+            PLUGIN_FILE_PATH=./${{ inputs.PLUGIN_NAME }}/${{ inputs.PLUGIN_NAME }}.php
+
+            # Replace beta with dev
+            RELEASE_NAME=${{ env.RELEASE_NAME }}
+            RELEASE_NAME=$(echo "$RELEASE_NAME" | sed 's/beta/dev/')
```
Comment (yotamselementor) [inline .github/workflows/release-dev-from-beta/action.yml:23]: Rename to `DEV_RELEASE_NAME`

File: .github/workflows/release-dev-from-beta/action.yml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    - name: Create Dev GitHub release
+      uses: softprops/action-gh-release@v1
+      with:
+        tag_name: ${{ env.DEV_RELEASE_NAME }}
+        files: ${{ env.DEV_BUILD_ZIP_FILE_PATH }}
+        body_path: ${{ env.CHANGELOG_FILE }}
+    - name: Publish to WordPress.org SVN
+      shell: bash
+      env:
+        PLUGIN_VERSION: ${{ env.PACKAGE_VERSION }}
```
Comment (yotamselementor) [inline .github/workflows/release-dev-from-beta/action.yml:50]:
```suggestion
        PLUGIN_VERSION: ${{ env.DEV_RELEASE_NAME }}
```

**Review by yotamselementor — CHANGES_REQUESTED** [review]

File: .github/workflows/release-dev-from-beta/action.yml (hunk 1 above)
Comment (maork-elementor) [reply .github/workflows/release-dev-from-beta/action.yml:18]: Done

File: .github/workflows/release-dev-from-beta/action.yml (hunk 2 above)
Comment (maork-elementor) [reply .github/workflows/release-dev-from-beta/action.yml:34]: Done

File: .github/workflows/release-dev-from-beta/action.yml (hunk 3 above)
Comment (maork-elementor) [reply .github/workflows/release-dev-from-beta/action.yml:23]: Done

File: .github/workflows/release-dev-from-beta/action.yml (hunk 4 above)
Comment (maork-elementor) [reply .github/workflows/release-dev-from-beta/action.yml:50]: Done

**Review by yotamselementor — APPROVED** [review]

**Pull request merged**